src/azure-cli/azure/cli/command_modules/acs/tests/latest/custom_preparers.py

```diff
@@ -3,31 +3,130 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from datetime import datetime
 import os
 
-from azure.cli.testsdk.preparers import ResourceGroupPreparer
+from azure.cli.testsdk.preparers import (
+    ResourceGroupPreparer,
+    VirtualNetworkPreparer,
+    KEY_RESOURCE_GROUP,
+    KEY_VIRTUAL_NETWORK,
+)
 
 
 class AKSCustomResourceGroupPreparer(ResourceGroupPreparer):
     def __init__(
-            self,
-            name_prefix='clitest.rg',
-            parameter_name='resource_group',
-            parameter_name_for_location='resource_group_location',
-            location='westus',
-            dev_setting_name='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_NAME',
-            dev_setting_location='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_LOCATION',
-            random_name_length=75,
-            key='rg'):
-        super(AKSCustomResourceGroupPreparer,
-              self).__init__(name_prefix, parameter_name,
-                             parameter_name_for_location, location,
-                             dev_setting_name, dev_setting_location,
-                             random_name_length, key)
+        self,
```

Comment:
I think this will have conflict with your other pr.

Reply:
changed due to strict style/format check, I will pay more attention to other prs.